src/itemviewformaction.cpp

```diff
@@ -235,13 +243,30 @@ bool ItemViewFormAction::process_operation(Operation op,
 	break;
 	case OP_BOOKMARK:
 		switch (bindingType) {
+		case BindingType::Bind:
+			if (args.empty()) {
+				this->start_bookmark_qna(
+					utils::utf8_to_locale(item->title()),
+					item->link(),
+					feed->title());
+			} else {
+				qna_responses = {
+					item->link(),
+					utils::utf8_to_locale(item->title()),
+					args.front(),
+					feed->title(),
+				};
+				finished_qna(OP_INT_BM_END);
+			}
+			break;
 		case BindingType::Macro:
 			qna_responses.clear();
 			qna_responses.push_back(item->link());
 			qna_responses.push_back(utils::utf8_to_locale(item->title()));
 			qna_responses.push_back(
 				args.size() > 0 ? args.front() : "");
 			qna_responses.push_back(feed->title());
+			finished_qna(OP_INT_BM_END);
```

Comment:
I don't think this was ever reported as an issue, but it looks like `bookmark` had no effect in a macro.